src/Common/Common.Api/Gateways/Entities/GatewayCluster.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
Need to add copyright headers to the new files:
/*
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License.
 */